packages/react-utilities/src/hooks/useCapture.ts

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+type UseCapture = {
```

Comment:
I think rather than having an object type for the params, it would be better to just do `useCapture(element: HTMLElement | undefined, pointerId: number | undefined)`. (`disabled` might be better handled by just passing undefined for the params in that case.)